src/scripts/main.js

```diff
@@ -1,3 +1,42 @@
 'use strict';
 
-// write code here
+const list = document.querySelectorAll('li');
+
+function sortList(element) {
```

Comment:
Try to name parameters more descriptively
```suggestion
function sortList(elements) {
```